### PR TITLE
Add test for downsampling `date_nanos` fields

### PR DESCRIPTION
**Pull request by csoulios**
Adding test to reproduce https://github.com/elastic/elasticsearch/issues/94085

### CODE REVIEW

**Comment by salvatore-campagna** [comment]
I added something similar in #94141

File: x-pack/plugin/rollup/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/rollup/50_date_nanos.yml [hunk 1]
```diff
@@ -0,0 +1,167 @@
+setup:
+  - skip:
+      version: " - 8.6.99"
```
Comment (salvatore-campagna) [inline x-pack/plugin/rollup/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/rollup/50_date_nanos.yml:3]: SHould this be 8.7.99...we are on 8.8.0 now. After backporting it we will change the version in the other pR right?

Comment (salvatore-campagna) [reply x-pack/plugin/rollup/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/rollup/50_date_nanos.yml:3]: I see you disabled the test using AwaitsFix.

**Review by salvatore-campagna — APPROVED** [review]

**Comment by csoulios** [comment]
Similar tests (and more) have been implemented in #94141. Closing this.

**Pull request closed without merging**